okhttp-logging-interceptor/src/main/java/okhttp3/logging/HttpLoggingInterceptor.java

```diff
@@ -192,7 +192,11 @@ public Level getLevel() {
         }
 
         logger.log("");
-        logger.log(buffer.readString(charset));
+        if (isPlainText(buffer)) {
+          logger.log(buffer.readString(charset));
+        } else {
+          logger.log("binary body omitted.");
```

Comment:
any way you can move the log so we only log a single line in this case, as on line 201 ? Maybe something like on 183?